.github/workflows/ci.yml

```diff
@@ -965,3 +965,59 @@ jobs:
           inputs: '{"version":"canary","environment":"staging","version_extra":"${{ needs.job_setup.outputs.branch_name }}"}'
           wait-for-completion-timeout: 25m
           wait-for-completion-interval: 30s
+
+  publish_admin_x_activitypub:
+    needs: [
+      job_setup,
+      job_lint,
+      job_unit-tests
+    ]
+    name: Publish @tryghost/admin-x-activitypub
+    runs-on: ubuntu-latest
+    if: always() && needs.job_setup.result == 'success' && needs.job_lint.result == 'success' && needs.job_unit-tests.result == 'success'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.12.1'
+
+      - name: Restore caches
+        uses: actions/cache@v4
+        with:
+          path: |
+            ${{ github.workspace }}/apps/admin-x-activitypub/node_modules
+          key: admin-x-activitypub-${{ github.sha }}
+          restore-keys: |
+            admin-x-activitypub-
+
+      - name: Install dependencies
+        run: yarn install --prefer-offline --frozen-lockfile
```

Comment:
You should be able to re-use our cache here: https://github.com/TryGhost/Ghost/blob/5579f03fd5fcc13c308f8c57c2e448dc10ad8e07/.github/workflows/ci.yml#L261-L264

It includes the `node_modules` for all packages but it's much faster than a `yarn install`